addons/sourcemod/scripting/scp_sf/scps/939.sp

```diff
@@ -66,7 +66,7 @@ public void SCP939_OnMaxHealth(int client, int &health)
 
 public void SCP939_OnSpeed(int client, float &speed)
 {
-	speed += (1.0-(Health[client]/HealthMax))*SpeedExtra;
+	speed += Pow((1.0-(Health[client]/HealthMax)), 2.0)*SpeedExtra;
```

Comment:
Ignoring `SpeedExtra` change, this would actually lowers speed as `1.0-(Health[client]/HealthMax)` is between 0.0 to 1.0, so squaring in this range would make result smaller. Honestly could just simply increase `SpeedExtra` and call it enough.

Reply:
and also messed up the calculations here being performed AFTER deducting from 1.0